packages/nimble-components/src/wafer-map/tests/wafer-map.stories.ts

```diff
@@ -32,26 +32,25 @@ interface WaferMapArgs {
 
 const getDiesSet = (
     setName: string,
-    sets: WaferMapDie[][]
+    _sets: WaferMapDie[][]
 ): WaferMapDie[] | undefined => {
     const seed = 0.5;
     let returnedValue: WaferMapDie[];
     switch (setName) {
-        case 'set1':
-            returnedValue = sets[0]!;
+        case 'randomDies10':
+            returnedValue = generateWaferData(10);
             break;
-        case 'set2':
-            returnedValue = sets[1]!;
+        case 'randomDies100':
+            returnedValue = generateWaferData(100);
             break;
-        case 'largeGoodSet':
-            returnedValue = generateWaferData(100, goodValueGenerator(seed))!;
+        case 'goodDies1000':
+            returnedValue = generateWaferData(1000, goodValueGenerator(seed))!;
             break;
-        case 'largeBadSet':
-            returnedValue = generateWaferData(100, badValueGenerator(seed))!;
+        case 'badDies10000':
```

Comment:
I love these new examples. I think the component looks really cool with tons of data.